src/core/osmLayer.js

```diff
@@ -108,7 +117,8 @@ geo.osmLayer = function (arg) {
   ////////////////////////////////////////////////////////////////////////////
   m_tileUrlFromTemplate = function (base) {
     return function (zoom, x, y) {
-      return base.replace('<zoom>', zoom)
+      return base.replace('{s}', m_this._getSubdomain(x, y))
```

Comment:
@jbeezley good point, will change it. I have seen more {} vs <>. Do you have any preference for one or the another?